From 85e2dc2bb0d2092b4614c802ec62cd8e05b90e29 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentfredric@gmail.com>
Date: Mon, 30 May 2016 11:39:14 +1200
Subject: [PATCH 9/9] Set V=0.28_01

---
 README             | 2 +-
 lib/Scope/Upper.pm | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/README b/README
index 5221b5a..e627c7b 100644
--- a/README
+++ b/README
@@ -2,7 +2,7 @@ NAME
     Scope::Upper - Act on upper scopes.
 
 VERSION
-    Version 0.28
+    Version 0.28_01
 
 SYNOPSIS
     "reap", "localize", "localize_elem", "localize_delete" and "WORDS" :
diff --git a/lib/Scope/Upper.pm b/lib/Scope/Upper.pm
index 87f4344..3adfcb3 100644
--- a/lib/Scope/Upper.pm
+++ b/lib/Scope/Upper.pm
@@ -11,13 +11,13 @@ Scope::Upper - Act on upper scopes.
 
 =head1 VERSION
 
-Version 0.28
+Version 0.28_01
 
 =cut
 
 our $VERSION;
 BEGIN {
- $VERSION = '0.28';
+ $VERSION = '0.28_01';
 }
 
 =head1 SYNOPSIS
-- 
2.8.3

